src/components/calcite-radio/calcite-radio.e2e.ts

```diff
@@ -0,0 +1,21 @@
+import { accessible, defaults, hidden, reflects, renders } from "../../tests/commonTests";
+
+describe("calcite-radio", () => {
+  it("renders", async () => renders("calcite-radio"));
+  it("is accessible", async () => accessible(`<calcite-radio></calcite-radio>`));
```

Comment:
You can pass the tag name directly since it doesn't need custom markup:

```ts
 it("is accessible", async () => accessible(`calcite-radio`));
```

This applies to all test helpers. :tada: